### ISSUE TITLE
Installing gr-ettus (rfnoc) using PYBOMBS has error in gnuradio install

### ISSUE THREAD

**Issue by mgarrett1955** [EttusResearch/gr-ettus#41]
Trying to install a clean RFNOC system on Ubuntu 18.4

I believe that the PYBOMBS script is pulling the gnuradio 3.8 branch down, which fails in the install phase with the following error. 

(This is the SAME error that one gets when installing gnuradio 3.8 from Pybombs, which I have posted to the pybombs issue list)

I believe that RFNOC currently requires gnuradio 3.7. 

Is the version forked on this site the correct repository to install gnuradio from, or should the gnuradio maint-3.7 repository be pulled.

I believe that the fix is to update the pybombs scripts to retrieve the correct version of gnuradio maint-3.7.

-- Python checking for numpy - not found
-- Configuring gnuradio-companion support...
-- Dependency ENABLE_GNURADIO_RUNTIME = ON
-- Dependency ENABLE_PYTHON = ON
-- Dependency PYTHON_MIN_VER_FOUND = TRUE
-- Dependency PYYAML_FOUND = TRUE
-- Dependency MAKO_FOUND = TRUE
-- Dependency PYGI_FOUND = TRUE
-- Dependency GTK_GI_FOUND = TRUE
-- Dependency CAIRO_GI_FOUND = TRUE
-- Dependency PANGOCAIRO_GI_FOUND = TRUE
-- Dependency NUMPY_FOUND = FALSE
CMake Error at cmake/Modules/GrComponent.cmake:75 (message):
user force-enabled gnuradio-companion but configuration checked failed
Call Stack (most recent call first):
grc/CMakeLists.txt:95 (GR_REGISTER_COMPONENT)

**Comment by mgarrett1955** [comment]
I believe this needs to be modified?

# Initializes a prefix with the necessary configuration to support RFNoC and RFNoC modtool
inherit: prefix
config:
    packages:
        uhd:
            gitbranch: master
            forcebuild: True
            vars:
              config_opt: " -DENABLE_RFNOC=ON "
        gnuradio:
            gitbranch: main-3.7  <<<---?
            forcebuild: True
        gr-ettus:
            gitbranch: master
            forcebuild: True
        uhd-fpga:
            gitbranch: master
depends:
- gnuradio
- uhd
- gr-ettus
- pyqt5
- uhd-fpga

**Comment by mgarrett1955** [comment]
Initializes a prefix with the necessary configuration to support RFNoC and RFNoC modtool
inherit: prefix
config:
    packages:
        uhd:
            gitbranch: master
            forcebuild: True
            vars:
              config_opt: " -DENABLE_RFNOC=ON "
        gnuradio:
            gitbranch: main-3.7 <<--?
            forcebuild: True

**Comment by primercuervo** [comment]
hi @mgarrett1955,

Thanks for reporting this. It does seem like the PyBOMBS recipes need a little tweak. I've created a PR for that. You can have a look at it and apply this patch in the meanwhile: https://github.com/EttusResearch/ettus-pybombs/pull/6

I've tested this with a freshly created prefix in Ubuntu 18LTS. Please do let us know if this helps!

**Comment by mgarrett1955** [comment]
I made the suggested change to rfnoc.lwr, and was able to successfully install the complete package, thank you!!

**Comment by mgarrett1955** [comment]
I installed it on another brand new clean Ubuntu 18.04 machine and it failed due to a missing dependancy.  I fixed it by installing the Ettus dependancies listed here, which I had done on the previous install: (unfortunately I nuke the error file on the install)

The error message showed a missign "windows.h"? In an obscure file... Ill see if I can find the error log again.

So there is a missing dependancy.

https://kb.ettus.com/Building_and_Installing_the_USRP_Open-Source_Toolchain_(UHD_and_GNU_Radio)_on_Linux

  sudo apt-get -y install git swig cmake doxygen build-essential libboost-all-dev libtool libusb-1.0-0 libusb-1.0-0-dev libudev-dev libncurses5-dev libfftw3-bin libfftw3-dev libfftw3-doc libcppunit-1.14-0 libcppunit-dev libcppunit-doc ncurses-bin cpufrequtils python-numpy python-numpy-doc python-numpy-dbg python-scipy python-docutils qt4-bin-dbg qt4-default qt4-doc libqt4-dev libqt4-dev-bin python-qt4 python-qt4-dbg python-qt4-dev python-qt4-doc python-qt4-doc libqwt6abi1 libfftw3-bin libfftw3-dev libfftw3-doc ncurses-bin libncurses5 libncurses5-dev libncurses5-dbg libfontconfig1-dev libxrender-dev libpulse-dev swig g++ automake autoconf libtool python-dev libfftw3-dev libcppunit-dev libboost-all-dev libusb-dev libusb-1.0-0-dev fort77 libsdl1.2-dev python-wxgtk3.0 git libqt4-dev python-numpy ccache python-opengl libgsl-dev python-cheetah python-mako python-lxml doxygen qt4-default qt4-dev-tools libusb-1.0-0-dev libqwtplot3d-qt5-dev pyqt4-dev-tools python-qwt5-qt4 cmake git wget libxi-dev gtk2-engines-pixbuf r-base-dev python-tk liborc-0.4-0 liborc-0.4-dev libasound2-dev python-gtk2 libzmq3-dev libzmq5 python-requests python-sphinx libcomedi-dev python-zmq libqwt-dev libqwt6abi1 python-six libgps-dev libgps23 gpsd gpsd-clients python-gps python-setuptools
